### PR TITLE
add four levels for each landmark

### PR DESCRIPTION
**Pull request by zxcv5678**
Screenshot 2024-09-23 at 14 37 32

### CODE REVIEW

File: src/components/LandmarkDrawer.tsx [hunk 1]
```diff
@@ -5,19 +5,41 @@ import {
...
 import { Landmark } from 'shared/Map';
+import React, { useState } from 'react';
+import { componentSpacing } from 'theme/metrics';
 
 export default function LandmarkDrawer ({ onClose, landmark }: { 
     onClose: () => void; 
     landmark: Landmark
   }) {      
-    return <Drawer isOpen onClose={onClose}>
+    const [value, setValue] = useState('');
```
Comment (weihanwang) [inline src/components/LandmarkDrawer.tsx:22]: Use LandmarkScore as the type of this value.

File: src/components/LandmarkDrawer.tsx [hunk 2]
```diff
@@ -5,19 +5,41 @@ import {
...
         <DrawerCloseButton /> 
         <DrawerHeader>{landmark.名称}</DrawerHeader>
-        <DrawerBody>{landmark.定义}</DrawerBody>
+        <DrawerBody display="flex" flexDirection="column" gap={componentSpacing}>
+          {landmark.定义}
+          <Text>你认为你的{landmark.名称}处于以下哪个阶段？（单选）</Text>
+          <RadioGroup onChange={setValue} value={value}>
+            <Stack direction="column">
+              {landmark.层级.map((level, index) => (
+                <Radio key={index} value={String(index)} alignItems="flex-start">
```
Comment (weihanwang) [inline src/components/LandmarkDrawer.tsx:35]: why flex-start?

Comment (weihanwang) [inline src/components/LandmarkDrawer.tsx:35]: does the value have to be a string and not a number?

File: src/components/LandmarkDrawer.tsx [hunk 3]
```diff
@@ -5,19 +5,41 @@ import {
...
+                  {level}
+                </Radio>
+              ))}
+            </Stack>
+          </RadioGroup>
+          <Button width="20%" alignSelf="center">提交</Button>
```
Comment (weihanwang) [inline src/components/LandmarkDrawer.tsx:41]: Do not manually specify style unless absolutely necessary. Why width="20%"?

Comment (weihanwang) [inline src/components/LandmarkDrawer.tsx:41]: use branding color for this button as it's a primary button.

File: src/components/LandmarkDrawer.tsx (hunk 2 above)
Comment (zxcv5678) [reply src/components/LandmarkDrawer.tsx:35]: UseRadioProps 接口指定了 value 的类型为 string：
Screenshot 2024-09-23 at 17 52 07
Screenshot 2024-09-23 at 17 54 26

Comment (zxcv5678) [reply src/components/LandmarkDrawer.tsx:35]: The paragraph is too long hence the button will appear in the middle. Which one do you prefer?
Screenshot 2024-09-23 at 17 49 39

**Review by weihanwang — APPROVED** [review]

**Pull request merged**